README.md

```diff
@@ -16,3 +16,5 @@ ABACUS (Atomic-orbital Based Ab-initio Computation at UStc) is an open-source pa
 
 # Online Documentation
 For detailed documentation, please refer to [our documentation website](https://abacus.deepmodeling.com/).
+
```

Comment:
It's not wiki, it is a github website.

Reply:
What's the recommended description of our Chinese doc page?

Reply:
It's a github webpage including tutorials and developer guides. You don't need to mention Chinese because in future we may have English version